runtime/vm/JFRConstantPoolTypes.cpp

```diff
@@ -536,6 +536,95 @@ VM_JFRConstantPoolTypes::addPackageEntry(J9Class *clazz)
 	return index;
 }
 
+U_32
+VM_JFRConstantPoolTypes::addPackageEntry(J9Module *fromModule, J9Package *package, BOOLEAN exported)
```

Comment:
Are there plans to call this method where exported could be false?

Reply:
Yes, thats a API point we plan to add in the future